admin/assets/javascripts/discourse/components/docker-manager/repo-status.hbs

```diff
@@ -1,62 +1,72 @@
-<tr>
+<tr class="repo {{if @repo.hasNewVersion 'new-version'}}">
```

Comment:
Can you please convert this component to a `.gjs` component while you are at it?